src/Collectors/DBQueryCollector.php

```diff
@@ -84,7 +80,7 @@ private function getQueryName(string $sql): string
 
             if (isset($query_type) && is_array($tables)) {
                 // Query type and tables
-                return $query_type . ' ' . join(', ', array_values($tables));
+                return $query_type.' '.join(', ', array_values($tables));
```

Comment:
Is this really in the coding standard? Readability is horrible...

Reply:
Yeah, I also think this is not good for readability. I'm going to tweak some rules.